### PR TITLE
4 notebooks ported from 4 DLAI agent short courses using Llama 3

### PR DESCRIPTION
**Pull request by jeffxtang**
# What does this PR do?

Fixes # (issue)

## Feature/Issue validation/testing

Please describe the tests that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
Logs for Test A

- [ ] Test B
Logs for Test B

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/facebookresearch/llama-recipes/blob/main/CONTRIBUTING.md#pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes?  
- [ ] Did you write any new necessary tests?

Thanks for contributing 🎉!

### CODE REVIEW

File: recipes/quickstart/agents/AI_Agentic_Design_Patterns_with_AutoGen_L4_Tool_Use_and_Conversational_Chess.ipynb [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+   "outputs": [],
+   "source": [
+    "!pip install chess\n",
+    "!pip install pyautogen"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a082a6dc-ceb1-4a3e-b3ae-afcb835de6da",
```
Comment (Karliz24) [inline recipes/quickstart/agents/AI_Agentic_Design_Patterns_with_AutoGen_L4_Tool_Use_and_Conversational_Chess.ipynb:37]: Traducción

**Comment by albertodepaola** [comment]
I know the idea is not to reproduce the full content of the courses, but might be good to contextualize the notebooks a bit more for them to make sense as a standalone content in the Llama recipes repo as well. Maybe a quick summary of the steps involved would be enough and how they complement the other samples in the repo.

**Comment by albertodepaola** [comment]
This looks good overall, and curious to see how it evolves as a standalone offering in the recipes as well!

**Review by albertodepaola — APPROVED** [review]
This looks good as an initial version of porting the content to the recipes. Curious to see how it evolves as a stand alone content that can be used with different demos in the repo.

**Pull request merged**